TakingITGlobal/commit2act
# Global style fixes due to restructure

This PR is to fix some missing dark mode styling that occurred as a result of reconfiguring the themes file. Took a bit of a walk to get there but the walk did involve some incremental fixes within the project mostly revolving around using the theme variables instead of hard coded hex codes. 

There is still a lot of work to be done in organizing the CSS still but I did the minimum I could to debug since it's out of scope for this budget. 

##Testing instructions: 
1. Pull branch
2. run `npm install`
3. run `npm start`
4. Evaluate the app visually around several pages to make sure all text and buttons are visible or at least consistent with the current live app. https://app.commit2act.org (note that I did change one colour border to better adhere to the theme)
